.github/workflows/dependencies.yml

```
@@ -51,6 +51,28 @@ jobs:
             type=raw,value=latest
             type=sha
 
+      - name: Detect dependency versions
+        id: versions
+        working-directory: dependencies
+        shell: bash
+        run: |
```

actionlint (1.7.7)

shellcheck reported issue in this script: SC2086:info:13:50: Double quote to prevent globbing and word splitting

(shellcheck)